src/main/java/net/snowflake/client/jdbc/SnowflakeDriver.java

```diff
@@ -61,6 +61,10 @@ private static void initializeArrowSupport() {
       disableArrowResultFormat = true;
       disableArrowResultFormatMessage = t.getLocalizedMessage();
     }
+    if ("true"
+        .equals(SnowflakeUtil.systemGetProperty("snowflake.jdbc.enable.illegalAccessWarning"))) {
```

Comment:
systemGetProperty cna return null. will it cause NPE?

Reply:
No, it's fine. We're only passing a null to the equals() method, so that will evaluate to false.